.gitignore

```diff
@@ -9,4 +9,5 @@ workers/unity/Assets/Plugins/Improbable/Generated/
 workers/unity/Assets/Plugins/Improbable/Generated.meta
 workers/unity/Assets/Plugins/Improbable/Sdk/Dll/Generated.Code.*
 workers/unity/Assets/Plugins/Improbable/EntityPrefabs/
+workers/unity/Assets/Plugins/Improbable/EntityPrefabs*
```

Comment:
This line is not necessary in the ignore file because the lines above and below will cover it